Sources/Client/Client/Client+Channel.swift

```diff
@@ -530,9 +530,7 @@ public extension Client {
     ///   - channel: a channel.
     ///   - completion: an empty completion block.
     @discardableResult
-    func deleteImage(url: URL,
-                     from channel: Channel,
-                     _ completion: @escaping Client.Completion<EmptyData> = { _ in }) -> Cancellable {
+    func deleteImage(url: URL, channel: Channel, _ completion: @escaping Client.Completion<EmptyData> = { _ in }) -> Cancellable {
```

Comment:
really don't want to make it an option, because it will lead us to make common Client request completion block optional (`Client+Request.swift` line 55):
```swift
func request<T: Decodable>(endpoint: Endpoint, _ completion: @escaping Completion<T>) -> Cancellable
```
In 1.x I mapped `EmptyData` (because we need codable empty data type) to `Void`, but I think we should avoid side effects and be clear what is happening.

Reply:
- What's wrong about having the completion optional everywhere, when it's actually really optional?
- I don't understand how's `EmptyData` related to this.

Reply:
If we want an optional completion block we should make in the general request (`Client+Request.swift`) is optional for all requests. We can change it ofc.

Reply:
OK, let's start talking about it and keep this as is for now.